example-proxy.ini

```diff
@@ -65,4 +63,18 @@ attribute = sAMAccountName
 # The timeout is specified in seconds.
 # This feature is EXPERIMENTAL.
 enabled = false
-timeout = 3
\ No newline at end of file
+timeout = 3
+
+[preamble-cache]
```

Comment:
Hm yeah, maybe the naming is not very intuitive. My line of thinking was:
We call the LDAP search performed by the application prior to a LDAP bind a "preamble". The "preamble cache"  remembers that a preamble happened for a specific timeframe. Specifically, it caches the DN of the queried user and the corresponding "app marker" (determined from the LDAP search filter, e.g. ``objectclass=App-<filter>``).
The realm mapping strategy ``preamble`` can then use the preamble cache to determine the realm for a given bind request. For that, the ``realm-mapping`` section contains a mapping from app markers to privacyIDEA realms.